### DISCUSSION TITLE
Super RTL [DE]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#918] [Q&A]
### Channel ID

RTLSuper.de

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/de.m3u#L382-L383
